.storybook/main.ts

```diff
@@ -27,5 +27,6 @@ const config: StorybookConfig = {
   docs: {
     defaultName: 'Documentation',
   },
+  staticDirs: ['../public'],
```

Comment:
Is this a necessary addition?

Reply:
I added it as it was instructed in the Storybook configuration. But it seems not to be required in this case.
It should be now removed.